easybuild/easyconfigs/z/zsh/zsh-5.8-foss-2019b.eb

```
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'zsh'
+version = '5.8'
+
+homepage = 'http://www.zsh.org/'
+description = "Zsh is a shell designed for interactive use, although it is also a powerful scripting language."
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+sources = [SOURCELOWER_TAR_XZ]
+source_urls = ['http://prdownloads.sourceforge.net/%(namelower)s']
+checksums = ['dcc4b54cc5565670a65581760261c163d720991f0d06486da61f8d839b52de27']
+
+dependencies = [
+    ('ncurses', '6.1'),
+]
+
+moduleclass = 'tools'
```

The following sanity checks are needed to pass the tests in github
```suggestion
sanity_check_paths = {
    'files': ['bin/zsh'],
    'dirs': ['lib/zsh/%(version)s', 'share'],
}

moduleclass = 'tools'
```

---

Very good, thank you.